### DISCUSSION TITLE
[Dropdown] search selection with allowAdditions behaviour

### DISCUSSION THREAD

**Discussion by bartocc** [fomantic/Fomantic-UI#1729] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hi,

With a `search selection dropdown` + allowAdditions, the first item is always selected after entering a search string that matches some items.

How can I select the user's addition and not the first item when hitting the enter key?

**Comment by ko2in** [comment]
I cannot reproduce as you said.
![Dropdown](https://user-images.githubusercontent.com/930315/97457245-f785d080-1967-11eb-88fa-ef7518006115.gif)

Here's fiddle (https://jsfiddle.net/8L3n02pw/). You can adjust the fiddle according to your use case.

**Comment by lubber-de** [comment]
You can reproduce in case you enter a value which is only a part of an existing value.
In your fiddle, try to add just "eng" . The first time "english" is preselected (thus selected when pressing enter). Only the second time it works, because there is no other fitting value anymore

![allowadditionshort](https://user-images.githubusercontent.com/18379884/97460528-1fac0a00-193d-11eb-8db9-da680df408c2.gif)

**Reply by bartocc** [reply]
this is the right reproduction @lubber-de 👍

Is there a way to configure the dropdown to select the additional value when the `Enter` key is pressed ?

**Comment by bartocc** [comment, marked as answer]
We finally fixed our UI/UX by using `hideAdditions=false`
